rep-I0001.rst

```diff
@@ -62,8 +62,7 @@ The industrial robot controller motion interface shall meet the following requir
  
 Design Assumptions
 =========
-The following assumptions are inherent in the client/server design:
- * All joints (regardless of group) shall be uniquely named.
+None at this time.
```

Comment:
Could you clarify why you removed that assumption? I can see why it would make sense to remove the *regardless of group* bit, but now it allows for multiple joints in the same group being identically named. Is that intentional?